trunk/src/app/srs_app_rtc_api.cpp

```diff
@@ -603,7 +605,16 @@ srs_error_t SrsGoApiRtcWhip::serve_http(ISrsHttpResponseWriter* w, ISrsHttpMessa
     // TODO: FIXME: Stop and cleanup the RTC session.
     if (r->method() == SRS_CONSTS_HTTP_DELETE) {
         string username = r->query_get("session");
+        string token = r->query_get("token");
+        if (token.empty()) {
+            return srs_error_new(ERROR_RTC_INVALID_SESSION, "token empty");
+        }
+
         SrsRtcConnection* session = server_->find_session_by_username(username);
+        if (token != session->token()) {
```

Comment:
Should it check session for null? `session->token()`

Reply:
Fixed